docusaurus/video/docusaurus/docs/api/recording/storage.mdx

```diff
@@ -84,9 +83,9 @@ call.start_recording(recording_external_storage= "my-s3")
 
 ```bash
 curl -X POST https://video.stream-io-api.com/video/external_storage?api_key=${API_KEY} \
-    -H "Authorization: ${JWT_TOKEN}" \
-    -H "Content-Type: application/json" \
+    -H "Authorization: ${TOKEN}" \
```

Comment:
`https://video.stream-io-api.com/api/v2/external_storage` isn't working so I left the old URL here